tests/test_mult_div.py

```diff
@@ -0,0 +1,37 @@
+# from myhdl import block, Signal, modbv, instance, always, delay
+# from riscv.md_constants import *
+# from riscv.mult_div import mult_div
+# from riscv.opcode_constants import XPR_LEN
+#
+#
+# @block
+# def test_mult_div():
+#     clock, reset, req_valid, req_ready, req_in_1_signed, req_in_2_signed = [Signal(False) for _ in range(6)]
+#     req_op = Signal(modbv(0)[MD_OP_WIDTH:])
```

Comment:
Why is this test commented? If the test is incomplete or doesn't pass yet, you can use py.test's xfail